src/data-structures/arrays/numbers/arraysOfNums.js

```diff
@@ -1,4 +1,4 @@
-export const generateArrayOfNums = (arrayCount, itemCount, range) => {
+const generateArrayOfNums = (arrayCount, itemCount, range) => {
     let arr = [];
```

Comment:
the naming doesn't reflect the function. 
the function creates an array of array containing numbers